src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_horizontal_layout.html.twig

```diff
@@ -145,7 +145,7 @@
   {%- set type = type|default('text') -%}
   <div class="input-group">
     {{- block('form_unit_prepend') -}}
-    {{- block('form_widget_simple') -}}
+    <div class="w-100">{{- block('form_widget_simple') -}}</div>
```

Comment:
Can work, but if you update the Twig Form theme you need to verify it works in places where it's used 😄 

Could you check
- another back-office page that does use this Input
- test your PR with this module https://github.com/PrestaShop/example-modules/tree/master/demosymfonyform it is an example module that adds a page with _every kind of input_ of the Form Theme

Reply:
I already have tested the rendering on another input with this type. I'll do it again with the module you suggested me ;)

Reply:
And actually I updated the theme because the problem is reproductible on every field that have this type

Reply:
I checked with de demosymfony form module and I didn't find any regression using this fix !